controls/pcidss_ocp4.yml

```diff
@@ -1799,17 +1840,24 @@ controls:
     * Something you are, such as a biometric.
   levels:
   - base
-  notes: ''
-  status: pending
+  notes: |-
+    The type of authenticators to be used (for example, password or passphrase,
+    token device or smart card, or biometrics) are also managed externally
+    to OpenShift by the identity provider
+  status: not appplicable
```

Comment:
There's a typo here, and it makes the build break:
```
 RuntimeError: Unable to parse controls from /content/controls/pcidss_ocp4.yml: The given status 'not appplicable' in the control 'Req-8.2' was invalid. Please use one of the following: ['pending', 'planned', 'not applicable', 'inherently met', 'documentation', 'partial', 'supported', 'automated'] 
 ```

Reply:
Ah my bad - I'll fix it right away